File: OneToOneCallSample/app/src/main/java/com/stringee/apptoappcallsample/common/CallStatus.java
package com.stringee.apptoappcallsample.common;

public enum CallStatus {
    INCOMING("Incoming"), CALLING("Calling"), RINGING("Ringing"), STARTING("Starting"), STARTED("Started"), BUSY("Busy"), ENDED("Ended");

    private final String value;

    CallStatus(String value) {
        this.value = value;
    }

    public String getValue() {
        return value;
    }
}

File: Kotlin-OneToOneCallSample/app/src/main/java/com/stringee/kotlin_onetoonecallsample/common/CallStatus.kt
package com.stringee.kotlin_onetoonecallsample.common


enum class CallStatus(val value: String) {
    INCOMING("Incoming"),
    CALLING("Calling"),
    RINGING("Ringing"),
    STARTING("Starting"),
    STARTED("Started"),
    BUSY("Busy"),
    ENDED("Ended")

}